playbooks/templates/artifact/index.html

```diff
@@ -399,7 +399,7 @@ <h3>Browse Artifacts</h3>
             {
               icon: 'graphic_eq',
               title: 'Build Summary Dashboard',
-              url: "http://rpc-repo.rackspace.com/rpcgating/buildsummary/"
+              url: "http://rpc-repo.rackspace.com/rpcgating/buildsummary/index.html#/job/" + this.job_name
```

Comment:
http://rpc-repo.rackspace.com/rpcgating/buildsummary/index.html#/job/PM_rpc-maas-master-xenial-ceph-deploy, for example, loads what appears to be a broken page (lots of 404s, etc.) Is this broken for anyone else?